[19.0][MIG] project_task_note: Migration to 19.0

jdidderen opened the pull request.

carolinafernandez-tecnativa committed:
[15.0][ADD] project_task_note: Add notes in project task
TT47764

[UPD] Update project_task_note.pot

carolinafernandez-tecnativa committed:
[MIG] project_task_note: Migration to 16.0

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: project-16.0/project-16.0-project_task_note
Translate-URL: https://translation.odoo-community.org/projects/project-16-0/project-16-0-project_task_note/it/

pilarvargas-tecnativa committed:
[IMP] project_task_note: pre-commit auto fixes

pilarvargas-tecnativa committed:
[MIG] project_task_note: Migration to 17.0

TT51644

david-banon-tecnativa committed:
[MIG] project_task_note: Migration to version 18.0

marcelsavegnago committed:
Added translation using Weblate (Portuguese (Brazil))

quirino95 committed:
[IMP] project_task_note: improved separation of internal notes section

pilarvargas-tecnativa committed:
[IMP] project_task_note: update maintainer

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: project-18.0/project-18.0-project_task_note
Translate-URL: https://translation.odoo-community.org/projects/project-18-0/project-18-0-project_task_note/

pedrobaeza committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: project-18.0/project-18.0-project_task_note
Translate-URL: https://translation.odoo-community.org/projects/project-18-0/project-18-0-project_task_note/es/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: project-18.0/project-18.0-project_task_note
Translate-URL: https://translation.odoo-community.org/projects/project-18-0/project-18-0-project_task_note/it/

jdidderen committed:
[MIG] project_task_note: Migration to 19.0

leemannd commented:
/ocabot migration project_task_note

cristina-hidalgo-tecnativa commented:
Code Review OK

Could you please merge the extra commits into the implementation and migration commits?

<img width="746" height="200" alt="imagen" src="https://github.com/user-attachments/assets/43c24766-9b85-45c5-9939-4507b2b49b49" />

<img width="721" height="204" alt="imagen" src="https://github.com/user-attachments/assets/38f5c793-a13c-4806-8a2f-5d311420adcb" />

victoralmau reviewed (approved):
Code review OK

jdidderen commented:
> Code Review OK
> 
> Could you please merge the extra commits into the implementation and migration commits?
> 
> <img alt="imagen" width="746" height="200" src="https://private-user-images.githubusercontent.com/87225960/540879306-43c24766-9b85-45c5-9939-4507b2b49b49.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzExNzk3MDMsIm5iZiI6MTc3MTE3OTQwMywicGF0aCI6Ii84NzIyNTk2MC81NDA4NzkzMDYtNDNjMjQ3NjYtOWI4NS00NWM1LTk5MzktNDUwN2IyYjQ5YjQ5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAyMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMjE1VDE4MTY0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYwNjQwNWNmYWVmZTcyNmRhNDk2YjA0NmI5YTAzM2MxNjkyMzExOWUyNTczZTU5OGE5NjlkYjRmY2I2MzhkMWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.ojVeFdehaWQ9Oie1VRw5ZcKJYzfayo98Qf0Q_f7ahO4"> <img alt="imagen" width="721" height="204" src="https://private-user-images.githubusercontent.com/87225960/540880817-38f5c793-a13c-4806-8a2f-5d311420adcb.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzExNzk3MDMsIm5iZiI6MTc3MTE3OTQwMywicGF0aCI6Ii84NzIyNTk2MC81NDA4ODA4MTctMzhmNWM3OTMtYTEzYy00ODA2LThhMmYtNWQzMTE0MjBhZGNiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjAyMTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwMjE1VDE4MTY0M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWViZThkZmYzZWM3MTc0OGIxNDkyOGZkODkwOGE2YmQ5ZGE4ZGM0NmNhNzg1N2NhZmI1OWI2ZDY2NzZhMTBiODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.oHqXVoIJONxc6Xu__2pNJ1CZ21-GbXmeNqqkdwUETu0">

I rebased the branch and squashed the commits

leemannd commented:
/ocabot merge nobump
